Foundation/FileManager+POSIX.swift

```diff
@@ -932,8 +932,14 @@ extension FileManager {
             if supportsStatx {
                 var statInfo = stat()
                 var btime = timespec()
-                guard _stat_with_btime(fsRep, &statInfo, &btime) == 0 else {
-                    throw _NSErrorWithErrno(errno, reading: true, path: path)
+                let statxResult = _stat_with_btime(fsRep, &statInfo, &btime)
```

Comment:
I dont think we need the `_SyscallResult` struct since `errno` as used in `_NSErrorWithErrono` should have the correct value anyway.